README.md

```diff
@@ -0,0 +1,81 @@
+[![Test / Cartesian](https://github.com/GridTools/gt4py/actions/workflows/test-cartesian.yml/badge.svg?branch=main)](https://github.com/GridTools/gt4py/actions/workflows/test-cartesian.yml)
+[![Test / Eve](https://github.com/GridTools/gt4py/actions/workflows/test-eve.yml/badge.svg?branch=main)](https://github.com/GridTools/gt4py/actions/workflows/test-eve.yml)
+[![Test / Next](https://github.com/GridTools/gt4py/actions/workflows/test-next.yml/badge.svg?branch=main)](https://github.com/GridTools/gt4py/actions/workflows/test-next.yml)
+[![Test / Storage](https://github.com/GridTools/gt4py/actions/workflows/test-storage.yml/badge.svg?branch=main)](https://github.com/GridTools/gt4py/actions/workflows/test-storage.yml)
+[![Code Quality](https://github.com/GridTools/gt4py/actions/workflows/code-quality.yml/badge.svg?branch=main)](https://github.com/GridTools/gt4py/actions/workflows/code-quality.yml)
+![license-badge](https://img.shields.io/github/license/GridTools/gt4py)
+
+
+# GT4Py: GridTools for Python
+
+## Description
+
+GT4Py is a Python library for generating high performance implementations
+of stencil kernels from a high-level definition using regular Python
+functions. GT4Py is part of the GridTools framework, a set of libraries
+and utilities to develop performance portable applications in the area
+of weather and climate.
+
+**NOTE:** this is a development branch for a new and experimental version
+of GT4Py working only with unstructured meshes and Python 3.10. The more
+stable version of GT4Py for Cartesian meshes lives in the `main` branch.
```

Comment:
I guess we should keep the README from main.

Reply:
Yes, let's keep all the docs from main and update them with `next` information in following PRs.